src/extensions/default/JavaScriptCodeHints/styles/brackets-js-hints.css

```diff
@@ -109,7 +109,7 @@
 
 .dark .brackets-js-hints.literal-hint {
     color: #bbb; /* dark grey */
-    font-family: SourceCodePro;
+    font-family: @sourceFontFamily;
```

Comment:
No variables in `.css` files, please.  Use a static `font-family` declaration here.